src/components/AudioPlayer/AudioPlayer.stories.mdx

```diff
@@ -0,0 +1,166 @@
+both of them.
+
+**This version will not be used for the playback of locally hosted audio files**.
+
+## Component Props
+
+<Canvas withToolbar>
+  <Story
+    name="AudioPlayer"
+    args={{
+      className: undefined,
+      descriptionText:
+        "Audio description lorem ipsum dolor simet. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Sed posuere consectetur est at lobortis.",
+      embedCode: `<iframe title="Libsyn Player" style="border: none" src="//html5-player.libsyn.com/embed/episode/id/18268511/height/90/theme/custom/thumbnail/yes/direction/backward/render-playlist/no/custom-color/87A93A/" height="90" width="100%" scrolling="no"  allowfullscreen webkitallowfullscreen mozallowfullscreen oallowfullscreen msallowfullscreen></iframe>`,
+      headingText: "Audio Title",
+      helperText: "Audio helper text lorem ipsum dolor simet.",
+      id: "audioplayer-id",
+      iframeTitle: "Libsyn Audio",
+      audioType: "libsyn",
+    }}
+  >
+    {(args) => (
+      <AudioPlayer
+        {...args}
+        embedCode={args.embedCode}
+        audioType={args.audioType && args.audioType}
+      />
+    )}
+  </Story>
+</Canvas>
+
+<ArgsTable story="AudioPlayer" />
+
+## Accessibility
+
+The `VideoPlayer` component renders an `iframe` element from `Libsyn`, `SoundCloud`, or `Spotify`.
+`iframe`s can be accessible if a `title` attribute is provided.
+The title must already be provided in the embed code value that is passed to the
+`embedCode` prop. A default one is provided but a custom one is recommended.
+
+Because these audios are hosted on a third-party platform, we cannot guarantee that the audio player will be accessible. Besides setting the `title` in the embedded `iframe`, there
+are _some ways_ we can reduce accessibility issues.
+
+- The player should not be set to auto-play when the page first loads.
+- The text in the heading, description and helper text element should be relevant and related to the embedded audio player.
+- The iframe title should describe in a few words the content of the embedded audio.
+- Consider using the description or helper text to add a link to a full transcript of the audio file. Refer to the link below for additional information.
+
+Resources:
+
+- [W3C Making Audio and Video Media Accessible](https://www.w3.org/WAI/media/av/)
+- [WebAIM Creating Accessible Frames and Iframes](https://webaim.org/techniques/frames/)
+- [MDN iframe: The Inline Frame element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe)
+- [Deque University Be Sure to Provide Titles for Iframes](https://dequeuniversity.com/tips/provide-iframe-titles)
+
+## Example Embed Code Snippets
+
+The examples below are shown with precise HTML formatting. However, often the
+embed code snippets provided by 3rd party services are delivered as one
+long string.
+
+### Libsyn
```

Comment:
The example iframe code you had before was fine to keep. I should've been clearer to say that the example canvas should be added in _addition_.

You can also add something like "The following is an example iframe code for Libsyn that would go into the `embedCode` prop."

```html
<iframe
  title="Libsyn Player"
  style="border: none"
  src="//html5-player.libsyn.com/embed/episode/id/18268511/height/90/theme/custom/thumbnail/yes/direction/backward/render-playlist/no/custom-color/87A93A/"
  height="90"
  width="100%"
  scrolling="no"
  allowfullscreen
  webkitallowfullscreen
  mozallowfullscreen
  oallowfullscreen
  msallowfullscreen
></iframe>
```

Reply:
You can add the other iframe examples for Spotify and Soundcloud.

Reply:
That makes sense. Updated.